docs/index.html

```diff
@@ -59,7 +61,7 @@
 
         <button class="accordion">Tournament</button>
         <div id="tournament-controls" class="panel">
-            <center><span class="note">Tournament not yet implemented</span></center>
+            <p style="text-align: center;"><span class="note">Tournament not yet implemented</span></p>
```

Comment:
Fair enough. I got a bit lazy about standards for a part that would be removed later.